scripts/sysman/common.yml

```diff
@@ -211,6 +211,10 @@ etors:
     - name: DEVICE_UUID
       desc: $s_uuid_t
       version: "1.7"
+    - name: MEMORY_EXP_STATE
```

Comment:
Could we name the structure type more specific to the feature.
Something like MEMORY_PAGE_OFFLINE_STATE_EXP.

This would help to extend MEMORY_STATE in the future.